pkg/syncer/metadatasyncer.go

```diff
@@ -3273,106 +3241,6 @@ func storagePolicyUsageCRSync(ctx context.Context, metadataSyncer *metadataSyncI
 	}
 }
 
-// startAvailabilityZoneCRInformer listens on changes to AvailabilityZone CR instances
```

Comment:
Mixed host reconciliation logic is moved from CSI layer to CNS layer, hence Informer to watch on AvailabalityZone CRs is no more required.